pl.art.lach.slawek.apps.v2blankbrowser.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "pl.art.lach.slawek.apps.V2BlankBrowser",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.38",
+  "sdk": "org.gnome.Sdk",
+  "command": "V2BlankBrowser",
+  "clear": [ "*.o"],
+  "build-options" : {
+    "arch": {
+      "x86_64": {
+      },
+      "x86": {
+      }
+    }
+  },
+  "modules": [
+  {
+    "name": "main",
+    "buildsystem" : "cmake",
+    "sources": [
+    {
+      "type": "dir",
+      "path": "."
+    }
+    ]
```

Comment:
This won't work either.

"type" "dir" is not accepted as it needs to have source in this repository.
You should have either an archive or git repository to pull the source from.